ansible/inventory

```diff
@@ -127,6 +127,11 @@ registry.proteus.ooni.io
 db-1.proteus.ooni.io
 proteus.ooni.io
```

Comment:
I tried to clean up all the tokens shared across all these 10 hosts of this group, but failed to do that _quickly_.
So, I just added another FIXME...